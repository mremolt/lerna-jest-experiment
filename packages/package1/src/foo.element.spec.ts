import { fixture, html } from '@open-wc/testing-helpers';
// @ts-ignore
import toDiffableHtml from 'diffable-html';

import { FooElement } from './foo.element';

describe('FooElement', () => {
  let subject: FooElement;

  beforeEach(() => {
    subject = new FooElement();
  });

  it('should build', () => {
    expect(subject).toBeDefined();
  });

  describe('render', () => {
    beforeEach(async () => {
      subject = await fixture(html` <package1-foo></package1-foo> `);
    });

    it('should render', () => {
      expect(toDiffableHtml(subject.shadowRoot!.innerHTML)).toMatchSnapshot();
    });
  });
});
