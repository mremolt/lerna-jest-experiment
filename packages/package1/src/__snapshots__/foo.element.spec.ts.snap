// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooElement render should render 1`] = `
"
Web Components are
<span>
  great
</span>
!
"
`;
